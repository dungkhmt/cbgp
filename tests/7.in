30 30
30 50
17 10
22 0
5 0
25 13
8 29
16 22
19 0
10 6
15 0
20 3
27 7
11 15
10 12
1 3
16 2
22 3
4 5
19 24
17 1
26 2
17 27
23 0
19 25
22 8
29 14
19 9
21 22
16 18
28 0
12 9
5 2
19 4
13 0
13 4
22 1
23 2
24 2
19 13
19 6
23 15
14 8
22 18
26 16
21 8
19 2
28 19
12 1
16 15
12 14
22 20
1
15 15 35 5 18 5 34 6 14 7 16 7 46 0 43 2 35 2 15 1 45 1 22 7 40 1 12 9 37 2 21 6 24 3 38 8 23 2 3 0 33 7 34 3 28 2 19 6 15 6 12 8 18 4 41 2 0 0 36 3 