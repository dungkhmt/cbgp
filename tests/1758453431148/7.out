(17, 10), (22, 0), (5, 0), (25, 13), (8, 29), (16, 22), (19, 0), (10, 6), (15, 0), (20, 3), (27, 7), (11, 15), (10, 12), (1, 3), (16, 2), (22, 3), (4, 5), (19, 24), (17, 1), (26, 2), (17, 27), (23, 0), (19, 25), (22, 8), (29, 14), (19, 9), (21, 22), (16, 18), (28, 0), (12, 9), (5, 2), (19, 4), (13, 0), (13, 4), (22, 1), (23, 2), (24, 2), (19, 13), (19, 6), (23, 15), (14, 8), (22, 18), (26, 16), (21, 8), (19, 2), (28, 19), (12, 1), (16, 15), (12, 14), (22, 20), 
numberIntersection=0 
minDistance=3.4246907579123573 
minAngle=0.07569285203385512 
sumAngle=40.86030610664219 
sumDistance=48687.47085905108 
minEdgeLength=3.605551275463989 
sumEdgeLength=1394.1001470408237 

0.0000000000 3.4246907579 0.0756928520 40.8603061066 48687.4708590511 3.6055512755 1394.1001470408
0: (48, 31),
1: (97, 70),
2: (96, 34),
3: (0, 47),
4: (90, 27),
5: (89, 31),
6: (118, 113),
7: (109, 80),
8: (110, 67),
9: (115, 57),
10: (114, 120),
11: (93, 44),
12: (115, 79),
13: (82, 22),
14: (111, 73),
15: (96, 41),
16: (102, 40),
17: (93, 80),
18: (107, 43),
19: (120, 4),
20: (65, 57),
21: (106, 64),
22: (108, 55),
23: (94, 37),
24: (98, 26),
25: (90, 22),
26: (102, 35),
27: (97, 80),
28: (91, 11),
29: (107, 71),
48 31 97 70 96 34 0 47 90 27 89 31 118 113 109 80 110 67 115 57 114 120 93 44 115 79 82 22 111 73 96 41 102 40 93 80 107 43 120 4 65 57 106 64 108 55 94 37 98 26 90 22 102 35 97 80 91 11 107 71 
0: (48, 31), 1: (97, 70), 2: (96, 34), 3: (0, 47), 4: (90, 27), 5: (89, 31), 6: (118, 113), 7: (109, 80), 8: (110, 67), 9: (115, 57), 10: (114, 120), 11: (93, 44), 12: (115, 79), 13: (82, 22), 14: (111, 73), 15: (96, 41), 16: (102, 40), 17: (93, 80), 18: (107, 43), 19: (120, 4), 20: (65, 57), 21: (106, 64), 22: (108, 55), 23: (94, 37), 24: (98, 26), 25: (90, 22), 26: (102, 35), 27: (97, 80), 28: (91, 11), 29: (107, 71), 
numberIntersection=0 minAngle=0.07569285203385512 sumAngle=40.8603061065984 minEdgeLength=3.605551275463989 sumEdgeLength=1394.1001470408232 minDistance=3.4246907579123573 sumDistance=48687.47085905131 
Time taken: 19.207 s
